### ISSUE TITLE
Send welcome SMS to users after signup

### ISSUE THREAD

**Issue by alexanderhoset** [IFRCGo/cbs#375]
After a user is signed up send them a welcome message defined in admin as a feedback message

This can be done by listening to "our own" user management events: **DataCollectorAdded** and **SaffUserAdded**
(More events can be added as #354 splits SaffUser to more roles )

After message is sent MobilePhoneNumberConfirmed  on the new user is set to true 

notes: 

- Default values for automatic reply messages #303 in admin is not done, so for now set a random message .
- The SMS gateway isn't implemented yet, I suggest we create an abstraction that we later integrate with the SMS gateway.

**Comment by karolikl** [comment]
@Davlind, can you confirm that this issue can be closed?

**Comment by Davlind** [comment]
Yes, it can be closed. Though, we should ensure that @alexanderhoset notes above have corresponding backlog items, as they're still valid
